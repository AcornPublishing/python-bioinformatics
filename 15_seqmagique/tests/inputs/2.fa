>SEQ0
GGGCATTGCTAACGATACGTACTCTATACGCCTAATGAATAAGGAATTTTGACTAGATAGAGTCAGACTCGAGATATCC
>SEQ1
TCCTAAATACTATCCTGCTGTGCAAGAGCCCCCACTCCCCACTGAGAATCTCCTGTGCG
>SEQ2
AATTTTTAGTCCTCCATATTTTCGCGGTGCTCAACCGAATCAAGTTCGTATCACGACAC
>SEQ3
GCAAGGTGGTCTGTCTTGATGTAGTTCTGGCAAGGTTTATGTATCGTGTTAACCTACTTTCTCAAGGCTTCCAA
>SEQ4
GAGATCGACCTACCTCGAGAGGAAGCTCCGCAAAATATTTAGGGAAAGG
