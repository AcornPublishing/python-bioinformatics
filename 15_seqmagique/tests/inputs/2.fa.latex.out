\begin{tabular}{lrrrr}
\hline
 name                &   min\_len &   max\_len &   avg\_len &   num\_seqs \\
\hline
 ./tests/inputs/2.fa &        49 &        79 &     64.00 &          5 \\
\hline
\end{tabular}
